Documentation.md

```diff
@@ -445,7 +449,7 @@ g.setOptions({
 
 
 Putting all this information together the final code to produce the chart included in the example index file provided is as follows:
-
+<span id='AdditionalHeaders'></span>
```

Comment:
why we need the span here? and why its outside of the "```html"  part?

Reply:
I added this span so that I can put a link in my new description. Clicking that link will take user to example code of additional headers

Reply:
good catch!